apprentice_teacher.rb

```diff
@@ -9,15 +11,6 @@ def initialize(options={})
     @phase = 3
   end
```

Comment:
Looks very similar to initialize for both `Teacher` and `Student`.  How can you refactor o that you don't have to include the age/name bits manually everywhere?